Containerfile.common

```diff
@@ -43,7 +43,13 @@ RUN if grep -qv "surface" <<< "${KERNEL_FLAVOR}"; then \
     /tmp/build-kmod-v4l2loopback.sh && \
     /tmp/build-kmod-wl.sh && \
     /tmp/build-kmod-xpadneo.sh && \
-    /tmp/build-kmod-xone.sh
+    /tmp/build-kmod-xone.sh && \
+    if grep -q "coreos" <<< "${KERNEL_FLAVOR}"; then \
+        curl -Lo /tmp/zfs-kmod-spec-in.patch "https://raw.githubusercontent.com/ublue-os/ucore-kmods/main/zfs-kmod-spec-in.patch" && \
```

Comment:
You shouldn't need this patch because we won't be building older versions

Reply:
The patch and logic was still in Ucore. The only reason I'm not just curling Ucore's install script is the rpm-ostree zlib-devel there causing issues with F40. My preference would be to just curl that script and not have to do the figure out which zfs version dance.

Reply:
Yep. Understood. We need to cleanup the ucore-kmods version of the script of the also.